website/docs/dapp/TutorialOverview.tsx

```diff
@@ -160,4 +160,53 @@ const DOBOVERVIEW: TutorialHeaderProps = {
   ],
 };
 
-export { TRANSFEROVERVIEW, WRITEOVERVIEW, TOKENOVERVIEW, DOBOVERVIEW };
+const LOCKOVERVIEW: TutorialHeaderProps = {
+  time: "10 - 15 min",
+  topics: [
+    {
+      label: "Full-Stack",
+      link: "/docs/tech-explanation/glossary#digital-object-dob",
```

Comment:
Should we remove the link for this? (maybe need to make the link as an optional parameter)

Reply:
> Should we remove the link for this? (maybe need to make the link as an optional parameter)

yea i forget to remove this one